fields/types/cloudinaryimage/CloudinaryImageField.js

```diff
@@ -292,7 +295,7 @@ module.exports = Field.create({
 		if (this.state.userSelectedFile || this.state.removeExisting) {
 			const value = this.state.userSelectedFile
 				? `upload:${this.state.uploadFieldPath}`
-				: '';
+				: `upload:remove`;
```

Comment:
@JedWatson for some reason when I set just `remove` I get a 400 POST error. It is expecting that `upload:` string... Something to do with express or something else? I haven't tracked down the problem yet.